parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetInputFormat.java

```diff
@@ -144,6 +144,11 @@
    */
   public static final String BLOOM_FILTERING_ENABLED = "parquet.filter.bloom.enabled";
 
+  /**
+    * Key to configure if off-heap buffer should be used for decryption
+    */
+  public static final String OFF_HEAP_DECRYPT_BUFFER_ENABLED = "parquet.decrypt.off-heap.buffer.enabled";
```

Comment:
Please document the new config parameter to the [README](https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/README.md)